cql_test.py

```diff
@@ -1219,12 +1219,6 @@ def _assert_logs_slow_queries_with_skinny_table(node, session):
         TestCQLSlowQuery._assert_logs(node, session, table,
                                       query="SELECT * FROM {} WHERE k >= 0 AND k < 100 ALLOW FILTERING",
                                       logged_query="SELECT \* FROM ks.{} WHERE k >= 0 AND k < 100")
-        TestCQLSlowQuery._assert_logs(node, session, table,
-                                      query="SELECT * FROM {} WHERE k <= 100 AND k > 0 ALLOW FILTERING",
-                                      logged_query="SELECT \* FROM ks.{} WHERE k > 0 AND k <= 100")
-        TestCQLSlowQuery._assert_logs(node, session, table,
-                                      query="SELECT * FROM {} WHERE k < 100 AND k >= 0 ALLOW FILTERING",
-                                      logged_query="SELECT \* FROM ks.{} WHERE k >= 0 AND k < 100")
```

Comment:
So technically, now we preserve the restrictions order, and we can fix the two `logged_query.`?
I am wondering why you decided to remove the two assertions instead. This test covered some wrong logging reported in CASSANDRA-15503. Maybe it is better to keep those assertions to ensure no future regressions?